src/state/registers.ts

```diff
@@ -5,6 +5,7 @@ import { Context, prompt, SelectionBehavior, Selections } from "../api";
 import { ArgumentError, assert, EditNotAppliedError, EditorRequiredError } from "../utils/errors";
 import { noUndoStops } from "../utils/misc";
 import type * as TrackedSelection from "../utils/tracked-selection";
+import { extensionName } from "../utils/constants";
```

Comment:
nit: please sort imports by import path (not obvious in the GitHub diff):

```suggestion
import { extensionName } from "../utils/constants";
import { ArgumentError, assert, EditNotAppliedError, EditorRequiredError } from "../utils/errors";
import { noUndoStops } from "../utils/misc";
import type * as TrackedSelection from "../utils/tracked-selection";
```